bokehjs/src/lib/models/widgets/tables/data_table.ts

```diff
@@ -247,19 +288,39 @@ export class DataTableView extends WidgetView {
       reorderable = false
     }
 
+    let frozenRow = -1
+    let frozenBottom = false
+    const frozenColumn = this.model.frozen_columns == null ? -1 : this.model.frozen_columns-1
+    if (this.model.frozen_rows != null) {
+      frozenBottom = this.model.frozen_rows < 0 ? true : false
+      frozenRow = Math.abs(this.model.frozen_rows)
+    }
```

Comment:
A few minor things:

1. Use snake case unless mandated by external factors (I will be enforcing this at some point).
2. `? true : false` could be removed.
3. `const {frozen_rows, frozen_columns} = this.model` if referencing them so many times.